### PR TITLE
Declare `@deprecated` on the stream wrapper functionality (will be removed in `3.0.0`)

### PR DESCRIPTION
**Pull request by sasezaki**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

### Description

* refs. #29 
* also see https://github.com/zendframework/zend-view/pull/34
* This PR aims to provide deprecation for v2 
* I use `@deprecated` phpdoc in order to deprecation.  https://discourse.laminas.dev/t/deprecation-policy-trigger-error-vs-deprecated/2435

### CODE REVIEW

**Comment by Ocramius** [comment]
@froschdesign added you as reviewer, mostly to check if any docs need changing

File: src/Stream.php [hunk 1]
```diff
@@ -14,6 +14,8 @@
  * written by
  *     Mike Naberezny (@link http://mikenaberezny.com)
  *     Paul M. Jones  (@link http://paul-m-jones.com)
+ *
+ * @deprecated will be removed in version 3
```
Comment (froschdesign) [inline src/Stream.php]:
```suggestion
 * @deprecated will be removed in version 3, only the normal PHP tags (<?php ?> and <?= ?>) should be used
```

Comment (sasezaki) [reply src/Stream.php]: @froschdesign Thanks suggestion. updated

**Review by Ocramius — APPROVED** [review]
Thanks @sasezaki!

**Pull request merged**